src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -137,20 +137,22 @@ Copyright (c) .NET Foundation. All rights reserved.
     <RuntimeFrameworkVersion Condition="'$(TargetLatestRuntimePatch)' != 'true' ">$(DefaultNetCorePatchVersion)</RuntimeFrameworkVersion>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(DisableImplicitFrameworkReferences)' != 'true'">
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(DisableImplicitFrameworkReferences)' != 'true'">
     <!-- This property is different than MicrosoftNETPlatformLibrary.  MicrosoftNETPlatformLibrary is
          used to trim the dependencies published with a framework-dependent app, and is overridden
          by ASP.NET packages.
-         
-         The ImplicitPlatformPackageIdentifier is the package that is actually implicitly
+
+         The ImplicitPlatformPackages is the list of packages that are actually implicitly
          referenced by the SDK, and is passed into the ResolvePackageAssets task
-         to verify that the restored version of the package matches the
-         RuntimeFrameworkVersion -->
-    <ImplicitPlatformPackageIdentifier>Microsoft.NETCore.App</ImplicitPlatformPackageIdentifier>
+         to verify that the restored versions of the packages matches the expected versions
+         -->
+    <ImplicitPlatformPackages Include="Microsoft.NETCore.App" Version="$(RuntimeFrameworkVersion)" />
```

Comment:
I would use `ExpectedVersion` instead of just `Verssion` as the metadata name here (and in the task that reads it).  That seems clearer to me.